test/common/grpc/json_transcoder_filter_test.cc

```diff
@@ -75,6 +76,17 @@ class GrpcJsonTranscoderConfigTest : public testing::Test {
       }
     }
   }
+
+  void stripImports(FileDescriptorSet& descriptor_set, std::string file_name) {
+    FileDescriptorProto file_descriptor;
+    for (auto& file : *descriptor_set.mutable_file()) {
```

Comment:
Nit: can you add a comment saying "filter down descriptor_set to only bookstore.proto but none of its dependencies"?